html-css/week1/index.html

```diff
@@ -37,43 +44,49 @@
       </div>
     </nav>
     <section id="profile">
-      <div class="section__pic-container">
-        <img src="./assets/profile-pic.png" alt="John Doe profile picture" />
-      </div>
+      
+        <img 
+        src="./assets/profile-pic.png" 
+        alt="Bogdan Dobinda's profile picture"
+        class="section__pic-container" />
+      
       <div class="section__text">
         <p class="section__text__p1">Hello, I'm</p>
         <h1 class="title">Bogdan Dobinda</h1>
         <p class="section__text__p2">Frontend Developer</p>
         <div class="btn-container">
-          <button
+          <a
             class="btn btn-color-2"
-            onclick="window.open('./assets/resume-example.pdf')"
+            href="./assets/resume-example.pdf"
           >
             Download CV
-          </button>
-          <button class="btn btn-color-1" onclick="location.href='./#contact'">
+        </a>
+          <a class="btn btn-color-1" href="./#contact">
             Contact Info
-          </button>
+          </a>
         </div>
         <div id="socials-container">
-          <img
-            src="./assets/linkedin.png"
-            alt="My LinkedIn profile"
-            class="icon"
-            onclick="window.open('https://www.linkedin.com/in/bogdyalex87/', '_blank')"
-          />
-          <img
-            src="./assets/github.png"
-            alt="My Github profile"
-            class="icon"
-            onclick="window.open('https://github.com/Bogdy87', '_blank')"
-          />
+          <a href="https://www.linkedin.com/in/bogdyalex87/" target="_blank">
+            <img
+              src="./assets/linkedin.png"
+              alt="My LinkedIn profile"
```

Comment:
The `alt` has to describe the graphic element and not the link, so in this case "LinkedIn icon" might be more fitting.